lib/shopify_api/resources/application_charge.rb

```diff
@@ -2,6 +2,12 @@ module ShopifyAPI
   class ApplicationCharge < Base
     undef_method :test
 
+    class << self
+      [:pending, :cancelled, :accepted, :declined].each do |status|
+        define_method(status) { (all || []).select { |c| c.status == status.to_s } }
```

Comment:
Thanks @awd.

I'm happy to do this if you really think this is necessary, but it seems to me that if the API is returning statuses in a different case than provided in the documentation, then the API is broken (or has been updated to a new major version, in which case all clients should have been notified and given time to make the appropriate changes).

I'm all for defensive programming, but I don't feel that should really extend to a client handling an API behaving outside of its spec.

My two cents - like I said, this is your call.